src/lib/pages/backoffice/Document/DocumentDetails/DocumentHeader.js

```diff
@@ -67,7 +67,7 @@ export class DocumentHeader extends Component {
           <DocumentAuthors
             authors={data.metadata.authors}
             hasOtherAuthors={data.metadata.other_authors}
-            prefix="by "
+            prefix="by"
```

Comment:
is there a white space left between `by` and the names of the authors after this change?